features/step_definitions/collection_steps.rb

```diff
@@ -134,3 +134,7 @@
 Then /^I should see a collection not found message for "([^\"]+)"$/ do |collection_name|
   step %{I should see /We couldn't find the collection(?:.+and)? #{collection_name}/}
 end
+
+Then /^the collection "(.*)" should be deleted/ do |collection|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.